kev/backends/dynamodb/db.py

```diff
@@ -0,0 +1,89 @@
+import decimal
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from botocore.exceptions import ClientError
+
+from kev.backends import DocDB
+from kev.exceptions import DocNotFoundError
+
+
+class DynamoDB(DocDB):
+
+    db_class = boto3.resource
```

Comment:
Yeah you should probably use to set self._db instead of just setting it to boto.resource('dynamodb') it should read self._db = self.db_class('dynamodb')